AppWidget/app/src/main/res/layout/widget_items_collection.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+  ~ Copyright (C) 2021 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    style="@style/Widget.AppWidget.AppWidget.Container"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:theme="@style/Theme.AppWidget.AppWidgetContainer">
+
+    <TextView
```

Comment:
I'd recommend re-using AppWidget/app/src/main/res/layout/widget_grocery_list_title_region.xml here and simply setting the title text when constructing the RemoteViews

Reply:
Changing the text of the TextView requires passing a variable to the include tag, which needs data binding. And I think it adds extract complexity than the benefit of reusing the layout.

Reply:
Re-opening

Reply:
This layout doesn't use the ImageButton.

Also, since This is a single TextView, I think having this TextView doesn't make the layout too complex.